tests/aggregation_regress/models.py

```diff
@@ -19,7 +19,7 @@ class Publisher(models.Model):
 class ItemTag(models.Model):
     tag = models.CharField(max_length=100)
     content_type = models.ForeignKey(ContentType, models.CASCADE)
-    object_id = models.CharField(max_length=24)
+    object_id = ObjectIdField()
```

Comment:
Did you forget to change this back? Does `CharField(max_length=24)` work just as well as `TextField`? I'd expect it to.

Reply:
Yes, I forgot.